robot/Cumulus/resources/NPSP.py

```diff
@@ -1223,6 +1223,31 @@ def select_value_from_dropdown(self,dropdown,value):
         self.selenium.get_webelement(locator).click()
         self.wait_for_locator('popup')
         self.npsp.click_link_with_text(value)
+
+    @capture_screenshot_on_error
+    def click_rd2_modal_button(self, name):
```

Comment:
please move the keyword to page object as this seems specific to RD2

Reply:
Removed